scripts/apps/publish/views/subscribers.html

```diff
@@ -280,7 +284,7 @@ <h3 class="modal__heading" translate ng-hide="subscriber._id">Add New Subscriber
         <div class="modal__footer">
             <button class="btn" ng-click="cancel()" translate>Cancel</button>
             <button class="btn btn--primary" ng-click="save()"
-                    ng-disabled="editForm.$invalid || (destinations.length === 0 && api_products.length === 0) || (destinations.length === 0 && products.length > 0) || !editForm.$dirty" translate>Save</button>
+                    ng-disabled="editForm.$invalid || !editForm.$dirty" translate>Save</button>
```

Comment:
Remove `$scope.enableSave` method and instead check if current data is different than saved data.